core/src/main/java/io/kestra/core/models/tasks/runners/types/ProcessTaskRunner.java

```diff
@@ -19,17 +20,65 @@
 import java.util.List;
 import java.util.Map;
 
+
+
 @Introspected
 @SuperBuilder
 @ToString
 @EqualsAndHashCode
 @Getter
 @NoArgsConstructor
-// all task runners are beta for now, but this one is stable as it was the one used before
-@Plugin(beta = true, examples = {})
 @Schema(
-    title = "A task runner that runs task as a process on the Kestra host",
-    description = "When the Kestra Worker that runs this process is terminated, the process will be terminated and the task fail."
+    title = "A task runner that runs a task as a sub-process of the Kestra process on the Kestra host.",
+    description = """
+        When the Kestra Worker that runs this process is terminated, the process will be terminated and the task fail.
+
+        Inside the task you can use:
+        - The `workingDir` Pebble variable or the `WORKING_DIR` environment variable to access the task working directory.
+        - The `outputDir` Pebble variable or the `OUTPUT_DIR` environment variable to access the output directory, each files located in this directory will be added to the task outputs."""
```

Comment:
```suggestion
        To access the task's working directory, use the `workingDir` Pebble expression or the `WORKING_DIR` environment variable.
        
        To generate output files within the task, you can leverage the `outputDir` Pebble expression or the `OUTPUT_DIR` environment variable.
        
        Note that when the Kestra Worker running this process is terminated, the task run will be marked as failed.
        """
```

Reply:
`workingDir` is a variable the expression would be `{{outputDir}}`. I can change to that if you want.